File: eshop-front/src/app/models/Order.ts
export class Order {
    userID: number;
    name: string;
    surname: string;
    productIDs: string;
    address: string;
    city: string;
    postalCode: string;
    paymentMethod: string;
    price: number;
    status: string;

    constructor(
        userID: number,
        name: string,
        surname: string,
        productIds: string,
        address: string,
        city: string,
        postalCode: string,
        paymentMethod: string,
        price: number,
        status: string) {
        this.userID = userID;
        this.name = name;
        this.surname = surname;
        this.productIDs = productIds;
        this.address = address;
        this.city = city;
        this.postalCode = postalCode;
        this.paymentMethod = paymentMethod;
        this.price = price;
        this.status = status;
    }
}

File: eshop-service/src/main/java/com/example/eshop/models/Order.java
package com.example.eshop.models;

import lombok.Data;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;

@Data
@Entity
@Table(name="Zamowienie")
public class Order {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer id;

    private Integer userID;
    private String productIDs;
    private String name;
    private String surname;
    private String address;
    private String city;
    private String postalCode;
    private String paymentMethod;
    private Double price;
    private String status;
}
